lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/string_view/TestDataFormatterStdStringView.py

```diff
@@ -78,14 +78,12 @@ def cleanup():
             "u32_string", type="std::u32string_view", summary='U"🍄🍅🍆🍌"'
         )
         self.expect_var_path("u32_empty", type="std::u32string_view", summary='U""')
-        self.expect_var_path(
-            "oops", type="std::string_view", summary='"Hellooo World\\n"'
-        )
 
         # GetSummary returns None so can't be checked by expect_var_path, so we
         # use the str representation instead
         null_obj = self.frame().GetValueForVariablePath("null_str")
-        self.assertEqual(null_obj.GetSummary(), "Summary Unavailable")
+        null_summary = null_obj.GetSummary()
+        self.assertTrue(null_summary == "Summary Unavailable" or null_summary is None)
```

Comment:
I don't have a preference here. I think not providing a summary at all should be as acceptable as saying "Summary Unavailable".

Reply:
I prefer being consistent with the other formatters and printing "Summary Unavailable". Not printing anything might get confused for "string is empty".